flocker/control/test/test_schemas.py

```diff
@@ -289,6 +289,55 @@
                 'remote_port': 65536
             }]
         },
+        # Volume with dataset_id of wrong type
+        {
+            'host': '192.168.0.3',
+            'image': 'postgres',
+            'name': 'postgres',
+            'volumes': [{'dataset_id': 123,
+                         'mountpoint': '/var/db'}],
+        },
+        # Volume with mountpoint of wrong type
+        {
+            'host': '192.168.0.3',
+            'image': 'postgres',
+            'name': 'postgres',
+            'volumes': [{'dataset_id': "x" * 36,
```

Comment:
Although I guess we decided not to require actual uuids... so it would be reason to say this should be fixed elsewhere first.